tests/unit-tests/devfile-page/Header.test.tsx

```diff
@@ -0,0 +1,184 @@
+import { Devfile } from 'custom-types';
```

Comment:
I dont see why `.tsx` would be required here

Reply:
it can't import the Header component without it being a tsx file

Reply:
or its more that the wrapper function uses react components and needs it to be a tsx file unless you want to do a bunch of workarounds